### PR TITLE
add page for jupyter-pyspark

### PR DESCRIPTION
**Pull request by ssb-jnk**
- Added new page for Jupyter-pyspark
- Added examples of configuring Spark resources (RAM, CPU)
- Added example showcasing how to use the Spark Jar-file extensions

### CODE REVIEW

File: dapla-manual/statistikkere/jupyter-pyspark.qmd [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+title: Jupyter-pyspark
+date-modified: last-modified
+---
+
+![](../images/dapla-lab-jupyter-pyspark-service.png){style="max-width: 50%; float: right;" fig-alt="Jupyter Service"}
+
+**Jupyter-pyspark** er en tjeneste på [Dapla Lab](./dapla-lab.html) som lar brukerne kode i [Jupyterlab](https://docs.jupyter.org/en/latest/). Tjenesten kommer Python, Pyspark og noen vanlige Jupyterlab-extensions ferdig installert. Målgruppen for tjenesten er brukere som skal skrive produksjonskode med Pyspark i Jupyterlab.
```
Comment (skykanin) [inline dapla-manual/statistikkere/jupyter-pyspark.qmd]:
```diff
- Tjenesten kommer Python, Pyspark og noen vanlige Jupyterlab-extensions ferdig installert.
+ I tjenesten kommer Python, Pyspark og noen vanlige Jupyterlab-extensions ferdig installert.
```

**Comment by akcssb** [comment]
You should add a post on the news-blog about the new article. I can also do it for you

**Review by skykanin — CHANGES_REQUESTED** [review]
One small comment, otherwise this looks good to me

File: dapla-manual/statistikkere/jupyter-pyspark.qmd (hunk 1 above)
Comment (ssb-jnk) [reply dapla-manual/statistikkere/jupyter-pyspark.qmd]: Fikset det, det var en glipp.

**Review by skars82 — APPROVED** [review]
Kjempebra

**Pull request merged**